### PR TITLE
Add Lattice1 Support and Bump BNC-Onboard to v1.19.0

### PR DESCRIPTION
**Pull request by MidnightOnMars**
**Background**
The Lattice1 is GridPlus' next generation Ethereum-focused programmable hardware wallet. It is an always-online device with physically separated secure compute and Linux general compute environments. For more information, see https://gridplus.io/

**About this PR**
This PR adds the Lattice1 as a wallet option for bnc-notify and bumps the Blocknative Onboard library version from v1.14.0 to v1.19.0. It also removes deprecated wallets no longer supported by the current version of the library (Dapper and Unilogin).

The Lattice1 can load smart contract ABIs so that interactions with your contracts are human-readable on the touchscreen which is drawn using the secure compute environment. This can help prevent man-in-the-middle attacks even when your computer is compromised. You can learn more about this functionality in [this blog post](https://blog.gridplus.io/readable-ethereum-transactions-a-new-standard-945c5e9ef2c7) and please let us know if you're interested in us preloading the Synthetix ABIs into all Lattice1s via an upcoming firmware release.

Thanks and please let us know if you have any questions!

EDIT: This is a revision of [an earlier PR I closed](https://github.com/Synthetixio/staking/pull/272) since Blocknative is now on to a newer version of Onboard and [this PR](https://github.com/Synthetixio/staking/pull/291) closed a few minutes ago by @clementbalestrat.

### CODE REVIEW

File: containers/Connector/config.ts [hunk 1]
```diff
@@ -20,6 +20,12 @@ export const initOnboard = (network: Network, subscriptions: Subscriptions) => {
 		walletSelect: {
 			wallets: [
 				{ walletName: 'metamask', preferred: true },
+				{
+					walletName: 'lattice',
+					appName: 'Synthetix',
+					rpcUrl: infuraRpc,
+					preferred: true,
```
Comment (clementbalestrat) [inline containers/Connector/config.ts:27]: @MidnightOnMars can you please remove `preferred: true`

File: containers/Connector/config.ts [hunk 2]
```diff
@@ -43,11 +49,9 @@ export const initOnboard = (network: Network, subscriptions: Subscriptions) => {
 					apiKey: process.env.NEXT_PUBLIC_PORTIS_APP_ID,
 				},
 				{ walletName: 'trust', rpcUrl: infuraRpc },
-				{ walletName: 'dapper' },
```
Comment (clementbalestrat) [inline containers/Connector/config.ts:46]: And put dapper back

**Review by clementbalestrat — CHANGES_REQUESTED** [review]

File: containers/Connector/config.ts (hunk 1 above)
Comment (MidnightOnMars) [reply containers/Connector/config.ts]: Updated to remove preferred status.

File: containers/Connector/config.ts (hunk 2 above)
Comment (MidnightOnMars) [reply containers/Connector/config.ts:46]: BlockNative Onboard removed support for Dapper, Unilogin, and Squarelink back in v1.18.0.

**Comment by clementbalestrat** [comment]
@MidnightOnMars looking good! Thanks for making the changes

**Pull request merged**